src/envoy/http/authn/filter_context_test.cc

```diff
@@ -37,7 +37,7 @@ class FilterContextTest : public testing::Test {
 
   // This test suit does not use headers nor connection, so ok to use null for
   // them.
-  StrictMock<FilterContext> filter_context_{nullptr, nullptr};
+  StrictMock<FilterContext> filter_context_{nullptr, nullptr, nullptr};
```

Comment:
I may forgot to clean this up, but I think we don't need StrickMock

Reply:
Done.